src/content-handlers/iiif/modules/uv-contentleftpanel-module/ContentLeftPanel.ts

```diff
@@ -754,26 +754,23 @@ export class ContentLeftPanel extends LeftPanel<ContentLeftPanelConfig> {
   selectCurrentTreeNodeByRange(): void {
     if (this.treeView) {
       const range: Range | null = this.extension.helper.getCurrentRange();
-      let node: TreeNode | null = null;
-
       if (range && range.treeNode) {
-        node = this.treeView.getNodeById(range.treeNode.id);
-      }
-
-      if (node) {
-        this.treeView.selectNode(<TreeNode>node);
-      } else {
-        this.selectTreeNodeByManifest();
+        const node = this.treeView.getNodeById(range.treeNode.id);
+        if (node) {
+          this.treeView.selectNode(node);
+        } else {
+          this.selectTreeNodeByManifest();
+        }
       }
     }
   }
 
   selectCurrentTreeNodeByCanvas(): void {
     if (this.treeView) {
       let node: TreeNode | null = null;
-      const currentCanvasTopRangeIndex: number = this.getCurrentCanvasTopRangeIndex();
-      const selectedTopRangeIndex: number = this.getSelectedTopRangeIndex();
-      const usingCorrectTree: boolean =
+      const currentCanvasTopRangeIndex = this.getCurrentCanvasTopRangeIndex();
```

Comment:
Yes, I removed the types because I didn't think the explicit type annotation was necessary but I'm not an expert so I can back to how it was to be just in case :)